Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+---
+versionFrom: 9.0.0
+versionTo: 10.0.0
+---
+
+# Block Grid
+
+`Alias: Umbraco.BlockGrid`
+
+`Returns: BlockGridModel`
+
+The **Block Grid** enables the editor to layout their content in the Umbraco backoffice. The content is made of blocks, which can contain simple or complex data.
+
+A Block takes full width with the default setup.
+There is two ways a Block can become smaller. When a Block is placed in a Area it will fit to the Areas width, or a Block can have one or more Column Span Options defined, A single Column Span Option will be used to define the Blocks width. With multiple options the Content Editor can scale the Block to fit one of these options.
+
+Additionally Blocks can be configured to span rows, this enables one Block to be placed next to a few rows containing other blocks.
+
+Blocks can nest other Blocks forming more complex or strict compositions. These compositions can be used as layout for other Blocks.
+To achieve nesting a Block must have one or more Areas defined. Each Area can contain one or more Blocks.
+
+The default provided layout mechanism is based on CSS Grid. This can be adjusted or replaced to achieve the right layout for your project.
+
+## Configure the Block Grid
+
+The Block Grid property editor is configured in the same way standard property editors, which is via the Data Types backoffice interface.
+
+To set up your Block Grid Editor property, follow these steps:
+
+1. Navigate to the **Settings** section in the Umbraco backoffice.
+2. Locate and right-click the **Data Types** folder.
+3. **Create a new Data Type**.
+4. Select **Block Grid** from the list of available property editors.
+
+You will see the configuration options for a Block Grid as shown below.
+
+![Block Grid - Data Type Definition](images/BlockListEditor_DataType.jpg)
+> TODO: screenshot
+
+The Data Type editor allows you to configure the following properties:
+
+- **Blocks** - Define the Block Types to be available for use in the property. Read more on how to set up Block Types below.
+- **Amount** - Sets the minimum and/or the maximum number of blocks that should be allowed at the root of the layout.
+- **Live editing mode** - Enabling this will enable you to see the changes you make as you are making them.
+- **Editor width** - Overwrite the width of the property editor. This field takes any valid CSS value for "max-width".
```

Comment:
@nielslyngsoe I think the example should be expanded on to give examples such as `100% or 800px or 100vw`